src/lib/pipeline.ml

```diff
@@ -225,16 +228,21 @@ module Full (Bfx: Extended_edsl.Semantics) = struct
   let hla fqs =
```

Comment:
might rename this "seq2hla" and make an equivalent one for Optitype. More importantly, need to add a Bfx.save for those optitypes, so they can be copied over into the final report.